app/models/classification.rb

```diff
@@ -32,6 +32,7 @@ class Classification < ApplicationRecord
   scope :read_only,  -> { where(:read_only => true) }
   scope :writeable,  -> { where(:read_only => false) }
 
+  # NOTE: roots does not work (that checks :parent_id => nil)
```

Comment:
Wonder if we can introduce parent_id => nil via a database migration, then parent_id => 0 will go away (it doesnt make sense anyway)

Reply:
@Fryguy Do you want me to change this to `parent_id => [nil, 0]` as we figure out this migration?

Do you want me to redefine `roots`?

Reply:
is this comment relevant to what remained of the PR?  what is `roots`?

Reply:
LGTM otherwise :+1: